napari/_tests/test_viewer.py

```diff
@@ -135,5 +135,58 @@ def test_changing_theme(viewer_factory):
     viewer.theme = 'light'
     assert viewer.palette['folder'] == 'light'
 
+
+    # Close the viewer
```

Comment:
I don't think we need the explicit close anymore and it is causing our tests to fail. @tlambert03 can you comment / advise?

Reply:
yeah remove the `viewer.window.close()` call.  See #2 in the [section on writing tests in the contributing guide](https://napari.org/docs/developers/CONTRIBUTING.html#writing-tests) for details.